### PR TITLE
[TAS-2134] 🐛 Fix email is empty in free purchase

### PR DESCRIPTION
**Pull request by nwingt**
In `data()`, `walletEmail` is `undefined`
https://github.com/likecoin/liker-land/blob/52a1737950ede7fd19de0ffe139e35f2002eadd4/src/pages/nft/claim/index.vue#L548-L559

Screenshot 2024-08-30 at 15 55 25

### CODE REVIEW

File: src/pages/nft/claim/index.vue [hunk 1]
```diff
@@ -709,6 +709,8 @@ export default {
     },
   },
   async mounted() {
+    this.claimingFreeEmail = this.walletEmail;
```
Comment (williamchong) [inline src/pages/nft/claim/index.vue:712]: set only if `free` is set? Not sure if we have such if case below for use

Comment (nwingt) [reply src/pages/nft/claim/index.vue]: Moved to `startFreePurchase()`

**Review by williamchong — APPROVED** [review]

**Review by AuroraHuang22 — APPROVED** [review]

File: src/pages/nft/claim/index.vue (hunk 1 above)
Comment (williamchong) [reply src/pages/nft/claim/index.vue]: Why did the watcher on `walletEmail` didn't work though

Comment (nwingt) [reply src/pages/nft/claim/index.vue]: my case is already logged in class page

**Pull request merged**